images/argocd-repo-server/tests/main.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "run" {
+  digest = var.digest
+  script = "docker run --rm $${IMAGE_NAME} --help"
```

Comment:
I was wondering when someone would do this :)

Reply:
Part of my desire to remove boilerplate was to make it easier to tell when this was possible. I'll probably make a pass at more of them in a bit.